### ISSUE TITLE
[Quest]The description about Retrieval Node in the quest is confusing

### ISSUE THREAD

**Issue by SihenZhang** [FTBTeam/FTB-Continuum#207]
* **Modpack Version**: 
1.2.1
* **Issue**: 
In the quest "Herer to Therer",  the description about Retrieval Node is "You can also create Retrieval Nodes that function similarly, but pull from inventories instead of push into."
But the Retrieval Node is input node. So the description is confusing.

**Comment by aaronhowser1** [comment]
The Retrieval Node pulls from inventories it's connected to with pipes and inserts items into the inventory it's placed on, while the Transfer Node pushes items into those inventories from the inventory it's placed on. Should I try to reword it to be more clear?

**Comment by SihenZhang** [comment]
I think yes. 
The description about Transfer Node is "pull from" **the inventory it's placed on**. So in my opinion, the description about Retrieval Node should also be about **the inventory it's placed on**.

**Comment by aaronhowser1** [comment]
Here's the new one, bold is what's changed:

Transfer Pipes and Nodes allow you to move items in a much easier and faster way than Conveyor Belts. Place the Node on the inventory/tank you wish to **extract** from and place a line of Transfer Pipes along the path you wish it to go, and connect it to the inventory/tank you wish to insert into. You can apply several upgrades and a Filter to Transfer Nodes by opening the Node's GUI and placing them inside.

You can also create Retrieval Nodes that function similarly, but **extract from inventories/tanks connected via pipes and inserts into the inventory/tank it's placed on**.

Items do not go through Nodes like other pipes. Instead, the Node slowly searches for an inventory that it can send items to, and then teleports the items there.

**Comment by tfox83** [comment]
Fixed in 1.4.0
